### DISCUSSION TITLE
Overriding vanilla items used in events causes those events to break

### DISCUSSION THREAD

**Discussion by Epsilore** [FakeFishGames/Barotrauma#14407] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

When a vanilla item like a duffel bag is overridden, it tends to refuse to cooperate with events, sometimes even deleting itself for no reason.

Duffel Bags: I was given a traitor mission to pick up a duffel bag and look inside. Looking inside did nothing.
ID Cards: I was given a traitor mission to steal someone's ID Card. I did so, held it in my inventory, tried wearing it, and nothing happened.

### Reproduction steps

1. Use a mod to override a vanilla item that is used in an event
2. Trigger the event
3. Try to interact with the overridden item

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! I unfortunately could not find a way to reproduce this. Here's how I attempted to test it:
1. Overrode idcard and duffelbag as follows:
```
  <Override>
    <Item name="" identifier="idcard" category="Equipment" Tags="smallitem,identitycard" cargocontaineridentifier="metalcrate">
      <PreferredContainer primary="crewcab"/>
      <Price baseprice="10">
        <Price storeidentifier="merchantoutpost" minavailable="4" />
        <Price storeidentifier="merchantcity" minavailable="6" />
      </Price>
      <InventoryIcon texture="Content/Items/InventoryIconAtlas.png" sourcerect="192,64,64,64" origin="0.5,0.5" />
      <Sprite texture="idcard.png" depth="0.5" sourcerect="0,0,16,16" />
      <Body width="16" height="12" density="10.05" />
      <IdCard slots="Card,Any" msg="ItemMsgPickUpSelect" />
    </Item>
    <Item name="" identifier="duffelbag" category="Equipment" tags="mobilecontainer,despawncontainer" fireproof="true" description="" impactsoundtag="impact_soft" scale="0.4">
      <Deconstruct time="5" />
      <Sprite texture="Content/Items/Jobgear/MiscJobGear.png" sourcerect="612,398,206,114" origin="0.5,0.56" depth="0.56" />
      <BrokenSprite texture="Content/Items/Jobgear/MiscJobGear.png" sourcerect="818,398,206,114" origin="0.5,0.56" depth="0.56" maxcondition="80" fadein="true" />
      <Body width="120" radius="45" density="20" friction="1.0" restitution="0.01" />
      <Holdable slots="RightHand+LeftHand" holdpos="0,-80" handle1="0,50" handle2="0,50" aimable="false" msg="ItemMsgPickUpSelect">
        <StatusEffect type="OnActive" targettype="This" Condition="-0.5" />
        <StatusEffect type="OnBroken" target="This">
          <DropContainedItems />
          <Remove />
        </StatusEffect>
      </Holdable>
      <ItemContainer capacity="20" keepopenwhenequipped="false" movableframe="true">
        <Containable items="smallitem,toolbelt,toolbox,bandolier" excludeditems="largemobilecontainer" />
      </ItemContainer>
    </Item>    
  </Override>
```
2. Triggered "theduffelbag" traitor event, which tasks you to look inside a duffel bag. Picking up the duffel bag seems to progress the event correctly.
3. Triggered "undercover", in which you need to steal an ID card. This too seemed to work with no issues.


It could be that this requires the overrides to do some specific kinds of changes to the item that make them incompatible with the events. Would it be possible to share us the XML of the overrides you're using, or if they're from some mods in the workshop, let us know which mods those are? Thanks and sorry for the inconvenience!
